tests/v2/validation/nodescaling/README.md

```diff
@@ -31,6 +40,76 @@ Typically, a cluster with the following 3 pools is used for testing:
   },
 }
   ```
-These tests are designed to accept an existing cluster that the user has access to. If you do not have a downstream cluster in rancher, you should create one first before running this test. 
 
-Scaling tests require that the given pools have unique, distinct roles and more than 1 node per pool. You can run a subset of the tests, but still need more than 1 node for the role you would like to run the test for. i.e. for `-run ^TestScaleDownAndUp/TestWorkerScaleDownAndUp$` you would need at least 1 pool with 2 or more dedicaated workers in it. The last node in the pool will be replaced. 
+See below some examples on how to run the node replacment tests:
+
+### RKE1
+`-run ^TestRKE1NodeScaleDownAndUp/TestEtcdScaleDownAndUp$`
+`-run ^TestRKE1NodeScaleDownAndUp/TestControlPlaneScaleDownAndUp$`
+`-run ^TestRKE1NodeScaleDownAndUp/TestWorkerScaleDownAndUp$`
+
+### RKE2 | K3S
+`-run ^TestNodeScaleDownAndUp/TestEtcdScaleDownAndUp$`
+`-run ^TestNodeScaleDownAndUp/TestControlPlaneScaleDownAndUp$`
+`-run ^TestNodeScaleDownAndUp/TestWorkerScaleDownAndUp$`
+
+## Scaling Existing Node Pools
+Similar to the `provisioning` tests, the node scaling tests have static test cases as well as dynamicInput tests you can specify. In order to run the dynamicInput tests, you will need to define the `scalingInput` block in your config file. This block defines the quantity you would like the pool to be scaled up/down to. See an example below:
+```json
+"scalingInput": {
+    "nodesAndRolesRKE1": [ 
+      {
+        "worker": true,
+        "quantity": 2,
+      },
+    ],
+    "nodesAndRoles": [
```

Comment:
needs a slight update after the machinePools / nodePools stuff went in 👀

Reply:
Sure, I'll update this, good catch

Reply:
Resolved